elastic/kibana
# [Security Solution] Update visualizationType

## Summary

origin issue: https://github.com/elastic/kibana/issues/139786

Steps to verify:

- On hosts / network / users page
- Trigger a chart action, and `Open in Lens`
- Please check the visualization you see is aligned with security solution.
- On Lens page's top right corner, click `Inspect`. If it is not a metric, make sure the column title exists.

Before:

![image](https://user-images.githubusercontent.com/59917825/187595410-781f29b7-b67c-48c0-8469-608de0d7a5c0.png)


https://user-images.githubusercontent.com/59917825/190333492-e131904d-12d5-4b4a-86c8-bfb2adb17292.mp4

After:
<img width="1679" alt="Screenshot 2022-09-21 at 21 53 31" src="https://user-images.githubusercontent.com/6295984/191608462-277d9ed2-333d-4651-970f-3402e8f0d031.png">
<img width="1674" alt="Screenshot 2022-09-21 at 21 54 01" src="https://user-images.githubusercontent.com/6295984/191608468-ef8d276a-af00-462a-9541-073f1983fab3.png">
<img width="1673" alt="Screenshot 2022-09-21 at 21 54 13" src="https://user-images.githubusercontent.com/6295984/191608472-31e9a36f-17ff-487a-8170-2b008fe6d5ce.png">



### Checklist

Delete any items that are not applicable to this PR.
